bin/appliance_console

```diff
@@ -320,6 +320,33 @@ Static Network Configuration
         end
         press_any_key
 
+      when I18n.t("advanced_settings.message_server")
```

Comment:
I think these should live under https://github.com/ManageIQ/manageiq-appliance_console/pull/130/files#diff-e8b26cb149425ac9294176fd67aa5744bcd030ff9ad96fe836bbdfa50607ce0dR449 with the existing external messaging configuration.

Reply:
Better link... https://github.com/ManageIQ/manageiq-appliance_console/blob/481d9b37c73a44471b13ce5c5d7a8fce00b7429d/bin/appliance_console#L449

Reply:
@bdunne Thank you. I will rearrange the menu items and group the message updates together with the current messaging prompts.